src/openrct2/network/Network.cpp

```diff
@@ -1502,6 +1506,9 @@ void Network::EnqueueGameAction(const GameAction *action)
 
 void Network::AddClient(ITcpSocket * socket)
 {
+    if (gConfigNetwork.pause_server && client_connection_list.size() == 0) {
```

Comment:
Since the server is able to pause and unpause at will, it'd be safer to check if the game is paused here.

Reply:
Good catch didn't know that there exists a function to get current game status 🙈 First time dealing with openrct2 sources, many thanks!